lib/tasks/repair_collections_refs.rake

```diff
@@ -0,0 +1,19 @@
+namespace :hydranorth do
+  desc 'Repair objects whose hasCollection property has been replaced with hasCollection_ref'
+  
+  task :repair_collections_refs => :environment do
+    GenericFile.all.each do |file|
+      response, xml = Hydranorth::RawFedora.get(file.id, 'fcr:export', format: 'jcr/xml')
+      next unless response == 200
+      
+      unless xml.xpath('//sv:property[@sv:name="ns002:hasCollection_ref"]').empty?
+        c = Collection.find(file.hasCollectionId.first)
+        
```

Comment:
Could we add a logging statement here, so that we'll know which items were actually modified? I'm thinking we should deploy this to test and run the rake task and then check modified items (or at least spot-check) before we run this on production data.